source/SIL.AppBuilder.Portal.Frontend/src/ui/styles/semantic-ui-overrides.scss

```diff
@@ -88,6 +88,14 @@ Inputs
   background-color: $checkbox-background-color;
 }
 
+.input {
+  position: relative;
+  svg, i {
+    position: absolute !important;
```

Comment:
a couple more `important!`s here an in the nex file.  necessary?

Reply:
yeah :(

Never using SemanticUI again.

Had I known about ShadowDom before this project, I would have mounted all the SemanticUI requiring things into a shadow dom, and use Material for everything else (or any other css framework that knows how to write css)